charts/rancher-webhook/values.yaml

```diff
@@ -6,6 +6,8 @@ image:
 global:
   cattle:
     systemDefaultRegistry: ""
+    psp:
+      enabled: true #PSPs enablement should default to true
```

Comment:
Do we need this comment?

Reply:
The comment was in the issue template and was adapted from the docs on this format. Given that it does not affect the functionality of this in anyway, I don't think it needs to change.